es/lessons/basics/collections.md

```diff
@@ -68,7 +72,7 @@ iex> tl [3.14, :pie, "Apple"]
 [:pie, "Apple"]
 ```
 
-Además de la funciones citadas, puedes usar el operador tubería `|`; veremos este patrón en futuras lecciones:
+Además de la funciones citadas, puedes usar [coincidencia de patrones](../pattern-matching/) y el operador tubería `|` para dividir una lista en cabeza y cola; veremos este patrón en futuras lecciones:
```

Comment:
It's been reported before, so after this is updated we can tick the task list here
https://github.com/doomspork/elixir-school/issues/608

I've been looking into it and there is no translation other than "operador cons"